request_handler.py

```diff
@@ -0,0 +1,186 @@
+# File: request_handler.py
+#
+# Copyright (c) 2017-2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+import json
+import os
+
+import encryption_helper
+import requests
+from django.http import HttpResponse
+
+import bmcremedy_consts as consts
+
+
+def handle_request(request, path_parts):
+    return BMCRequestHandler(request, path_parts).handle_request()
+
+
+def _get_dir_name_from_app_name(app_name):
```

Comment:
There is a function for this in phantom `app_name_to_dir_name` in `app_utils`. Should be available to connectors

Reply:
I checked, we will use app_utils function directly in connector file.

Reply:
I already did the suggested changes. Could you please review and resolve this thread.